docker-compose.local.yml

```diff
@@ -0,0 +1,87 @@
+#
+# docker-compose file used ONLY for local development.
```

Comment:
Maybe let's call it `dev-full` for clarity? `local` isn't that obvious 😅

Reply:
100%